src/odemis/util/driver.py

```diff
@@ -124,27 +124,48 @@ def estimateMoveDuration(distance, speed, accel):
     acceleration. It considers that the speed curve of the move will follow
     a trapezoidal profile: first acceleration, then maximum speed, and then
     deceleration.
-    distance (0 <= float): distance that will be traveled (in m)
-    speed (0 < float): maximum speed allowed (in m/s)
-    accel (0 < float): acceleration and deceleration (in m/s²)
+
+    :param distance: (0 <= float) distance that will be traveled (in m)
+    :param speed: (0 < float) maximum speed allowed (in m/s)
+    :param accel: (0 < float) acceleration and deceleration (in m/s²)
```

Comment:
```suggestion
    :param accel: (0 < float) acceleration (in m/s²)
```

Reply:
That's also deceleration. Why would you remove it?

Reply:
`accel` parameter does not accept negative values (i.e. deceleration), so just have acceleration in the docstring?

Reply:
How about: 
```python
:param accel: (0 < float) acceleration of the move, which is equal to the deceleration (in m/s²)
```

Reply:
Works!